frame/balances/src/tests.rs

```diff
@@ -48,14 +34,13 @@ macro_rules! decl_tests {
 		};
 		use pallet_transaction_payment::{ChargeTransactionPayment, Multiplier};
 		use frame_system::RawOrigin;
+		use frame_system as system;
 
 		const ID_1: LockIdentifier = *b"1       ";
 		const ID_2: LockIdentifier = *b"2       ";
 
-		pub type System = frame_system::Module<$test>;
-		pub type Balances = Module<$test>;
-
-		pub const CALL: &<$test as frame_system::Config>::Call = &$crate::tests::CallWithDispatchInfo;
+		pub const CALL: &<$test as frame_system::Config>::Call =
+			&Call::Balances(pallet_balances::Call::transfer(0, 0));
```

Comment:
This now needs to be an instance of the actual `Call` from `construct_runtime`, so just constructing a dummy `transfer`. Any better idea?

Reply:
I don't have a better idea, but a comment in the code would be welcome. :)